### ISSUE TITLE
Different Web UI used for LAN and WAN access?

### ISSUE THREAD

**Issue by anohren** [Secretmapper/combustion#55]
I installed the UI by adding the following to `/etc/systemd/system/transmission.service.d/override.conf`:

```
[Service]
Environment="TRANSMISSION_WEB_HOME=/home/username/downloads/transmission-web-ui/combustion/combustion-release"
```

This worked well when I connected locally, i.e. `rpi.local:9091/transmission/web`, but I'm being served the old UI whenever I try to access it over WAN in the form `example.com:12345/transmission/web` which is NATed to the same local address and port. 

Did I miss a step in my installation?

**Comment by ozamorowski** [comment]
I thing it's transmission problem, not the UI itself. Copy files to default UI folder `~/.local/share/transmission/web/`  and everything should work fine.

**Comment by anohren** [comment]
Thanks for the suggestion! I was trying to avoid overwriting the existing files by setting the environment, but I guess it can't be helped.

Weird behavior of Transmission...

**Comment by anohren** [comment]
Yeah you were right. Symlinking `/usr/share/transmission/web -> /home/username/downloads/transmission-web-ui/combustion/combustion-release/` worked fine eventually.
